teachopencadd/talktorials/001_query_chembl/talktorial.ipynb

```diff
@@ -996,22 +1046,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 25,
+   "execution_count": 43,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Number of bioactivities before ChEMBL 27 filter: 7178\n",
-      "Number of bioactivities after ChEMBL 27 filter: 7178\n"
+      "Number of bioactivities queried for EGFR in this notebook: 7178\n",
+      "Number of bioactivities after ChEMBL 27 intersection: 7178\n"
      ]
     }
    ],
    "source": [
-    "print(f'Number of bioactivities before ChEMBL 27 filter: {bioactivities_df.shape[0]}')\n",
+    "print(f'Number of bioactivities queried for EGFR in this notebook: {bioactivities_df.shape[0]}')\n",
+    "# Get the intersection between the current and the frozen ChEMBL version\n",
     "bioactivities_df = bioactivities_df[bioactivities_df.activity_id.isin(bioactivity_ids_chembl_27)].copy()\n",
```

Comment:
Dominique suggested that we use bioactivities_df["activity_id"] instead of "bioactivities_df.activity_id", so I just change it and the following.